docs/data/articles.yml

```diff
@@ -64,8 +64,7 @@ search:
   change-the-storefront-search-filters-ui.html.md: /articles/change-the-storefront-search-filters-ui.html.md
 cart-and-checkout:
   checkout.html.md: /articles/checkout.html.md
-  configure-a-payment-gateway.html.md: /articles/configure-a-payment-gateway.html.md
-  integrate-a-payment-gateway.html.md: /articles/integrate-a-payment-gateway.html.md
+  implement-the-credit-card-tender-type.html.md: /articles/implement-the-credit-card-tender-type.html.md
```

Comment:
could we shorten this to "Implement Credit Card Tenders"?

Reply:
When all the docs in this set are finished the titles will be

* Payment Tender Types
* Implement the Credit Card Tender Type
* Implement a Primary Tender Type
* Implement an Advanced Payment Tender Type

with the first doc providing the context for the three that follow.

All four describe the "Tender Type" extension point provided by the platform.